src/Joystick/JoystickAndroid.cc

```diff
@@ -285,7 +285,7 @@ bool JoystickAndroid::init() {
     return true;
 }
 
-static const char kJniClassName[] {"org/mavlink/qgroundcontrol/QGCActivity"};
+static const char kJniClassName[] {"org/mavlink/qgroundcontrol/QGCUsbSerialManager"};
```

Comment:
Is this correct. Seems strange?

Reply:
Yeah, it's because all the serial methods were moved to a separate class "QGCUsbSerialManager" to organize the code a bit.